main.js

```diff
@@ -0,0 +1,54 @@
+const fs = require('fs');
+const readline = require('readline');
+var filaName = 'users.csv'
+const fileStream = fs.createReadStream(filaName);
+
+var interfaceReader = readline.createInterface({
+    input: fileStream
+});
+
+
+async function readFromFile(){
+    users = [] ;
+    for await (const line of interfaceReader) {
+       
+       const temp = line.split(",")
+       users.push(temp)
+       //console.log(temp)
+      
+       
+    }
+    return users;
+}
+
+async function writeOnFile(users){
+    let csv = "";
+    for (let index = 1; index < users.length; index++) {
+        csv += users[index]
+        csv+="\n"
+    }
+    fs.writeFileSync("write.csv", csv);
+    return users
+
+}
+
+async function writeOnJsonFile(users){
```

Comment:
It is nice to move these utils to a separate folder.

Reply:
What did you mean by that ?